corehq/apps/app_manager/static_strings.py

```diff
@@ -61,7 +61,6 @@
     gettext_noop('CommCare LTS'),
     gettext_noop('CommCare Sense'),
```

Comment:
this is J2ME only also, isn't it?

Reply:
It sounds like it. Not sure how to check that though. Any suggestion on how to be sure?

Reply:
I was basing that assumption on this removed setting:
https://github.com/dimagi/commcare-hq/pull/32845/files#diff-d70813994fe014b9b6b9bd291dc0f6d279092329737c86d3b650dd7bf528b2c8L213
I believe these `static_strings` are here so those strings are marked for translation, and those translations are then used for other places those strings are shown, which is the yaml files.

It looks like this is where that happens:
https://github.com/dimagi/commcare-hq/blob/d7661872932367ce9e19cda52877ad2b914c82b1/corehq/apps/app_manager/commcare_settings.py#L54-L56
That is, since the yaml files aren't python or javascript, the scripts that gather translatable UI strings won't find them.  This is a workaround, using placeholder strings that correspond to what's in the yaml files.

Reply:
@esoergel Deleted another 2000 lines of translations.